engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheHashingJobActor.scala

```diff
@@ -274,7 +275,7 @@ object CallCacheHashingJobActor {
           // If the current batch is empty, we got a partial result, take the first BatchSize of the list
           if (updatedBatch.isEmpty) {
             // hashResult + fileHashResults.take(BatchSize - 1) -> BatchSize elements
-            val partialHashes = NonEmptyList.of[HashResult](hashResult, fileHashResults.take(BatchSize - 1): _*)
+            val partialHashes = NonEmptyList.of[HashResult](hashResult, fileHashResults.take(batchSize - 1): _*)
```

Comment:
@cjllanwarne If you're talking about this `- 1`, it's not to make room for the base hash, it's just because this function is called when we have a new hash (`hashResult`). So to create a bucket of size `batchSize` we take this new hash plus the last `batchSize - 1` hashes

Reply:
Although adding the base hash as a filter on top of the bucket could be interesting to try out